src/test/java/CarTest.java

```diff
@@ -0,0 +1,30 @@
+import RacingCar.Car;
+import RacingCar.MovableStrategy;
+import RacingCar.NonMovableStrategy;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+    Car car = new Car();
+
+    @Test
+    @DisplayName("3번 시도시, 자동차는 전진한 결과는  이다(전진 못함)")
+    public void ThreeTryNonMoveTest() {
+        for (int i = 0; i < 3; i++) {
+            car.move(new NonMovableStrategy());
+        }
+        assertThat(car.getCurPosition()).isEqualTo("");
+    }
+
+    @Test
+    @DisplayName("3번 시도시, 자동차가 전진한 결과는 ---이다")
+    public void ThreeTryThreeMoveTest() {
+        for (int i = 0; i < 3; i++) {
+            car.move(new MovableStrategy());
+        }
```

Comment:
인터페이스를 사용한 설계를 한 덕분에 랜덤한 숫자에 상관없이
프로덕션 코드를 테스트해 볼 수 있도록 변경이 되었어요!!!👏👏👏
아래처럼 말이지요
```suggestion
        for (int i = 0; i < 3; i++) {
            car.move(new MoveStrategy() {
                @Override
                public boolean isMove() {
                    return true;
                }
            });
        }
```
이렇게 변경할수 있구요,
이걸 람다를 사용하면 다시 아래처럼 간단하게 표현할 수 있습니다.
```java
        for (int i = 0; i < 3; i++) {
            car.move(() -> true);
        }
```
이렇게 인터페이스를 메소드의 인자로 사용하게 되면
테스트에서 인터페이스의 내용을 구현하여 내가 행동을 제어할 수 있어서
원하는 테스트 결과값을 얻을 수 있답니다😁